third_party/xla_client/pjrt_computation_client.h

```diff
@@ -146,7 +145,11 @@ class PjRtComputationClient : public ComputationClient {
   };
 
  private:
-  std::unique_ptr<PjRtClient> client_;
+  std::shared_ptr<PjRtClient> client_;
+  std::unordered_map<std::string, xla::PjRtDevice* const> string_to_device_;
+  std::shared_ptr<std::vector<std::string>> replication_devices_;
+
```

Comment:
nit. I think we could move `PjRtData` and `PjRtComputation` to private member section?

Reply:
Good catch. I was following ComputationClient, but users should not be using PjRtData/Computation directly, so these should be private instead. I'll move them in a follow-up PR.